### DISCUSSION TITLE
corect link of images

### DISCUSSION THREAD

**Discussion by Bilenki** [mmistakes/minimal-mistakes#2999] [Q&A] [answered]
## Environment

- Minimal Mistakes version: up-to-date
- Jekyll version: 4.0.1
- Git repository URL: github Enterprise

## Expected behavior

I would like to use the standard md syntax for images.
Example: ![](IMAGEPATH)

## Steps to reproduce the behavior

My images won't displayed in the github page, when I use the the following syntax ![](IMAGEPATH)

I need to add {{ site.url }}{{ site.baseurl }} in front of it. It looks like this then: ![]({{ site.url }}{{ site.baseurl }}/IMAGEPATH)
Disadvantage is that my images won't displayed in the normal MD file, only in the page

Could you please explain it to me? WHat are the advantages? How can I fix it?

**Comment by mmistakes** [comment, marked as answer]
This isn't unique to this theme. If you're building a Jekyll site in a subfolder that needs to use `baseurl`, then all your internal links and asset paths need to be prepended with `{{ site.baseurl }}` or use the `relative_url` filter.

You can omit `site.url` from the path. If your site is just `somedomain.com` then you could get away with `![]/your-image-path`, but due to using `baseurl` it needs to be:

```markdown
![]({{ site.basurl }}/your-image-path)
```

Or

```markdown
![]({{ 'your-image-path' | relative_url }})
```
